src/Mesh/babylon.geometry.ts

```diff
@@ -267,7 +267,7 @@
         }
 
         public getTotalVertices(): number {
-            if (!this.isReady()) {
+            if (this.isReady() === false) {

```

Comment:
is this a real gain?

Reply:
Alone no. There is a number of functions which are called thousands of times each frame. With a scene with LOTS of meshes, these small optimizations to if-statements within the functions that are called a lot do have a measurable impact. Alone this does not make enough of an impact to measure, but when this type of optimization is made to a number of places within code that is called a lot, yes, quite a bit. You will see I made this type of change in a number of functions. I was using Chrome's tools to profile frame time and then did this type of quick optimization before moving on to other optimizations I am submitting with this PR.

I measured the difference a few weeks ago when I was first working on this, but sadly did not save the results. I saw about a 8 to 15% decrease per frame when all of the small if-statement optimizations were added up. And when trying to hit 60 fps per second every millescond matters it was significant.

Reply:
For instance running this: https://jsperf.com/vs-false/1
will get slightly better result with original code:
![image](https://user-images.githubusercontent.com/1306056/34463547-1ffd5df8-ee14-11e7-82ca-b7d201336675.png)

Reply:
I was doing the same thing, and see similar. Here is the test I was running:
https://jsperf.com/strict-equal-vs-generic-if/1

The winner is very close on my computer in both Chrome and Firefox.

Reply:
yes so this tends to confirm my thought that we may not need this change